docs/source/release/index.rst

```diff
@@ -12,6 +12,8 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
 
+* :bug:`2829` Fix .drop(fields). The argument can now be either a list of strings or a string.
+  In the latter case, the given string is transformed into one element list.
```

Comment:
These notes are for Ibis users. I think they'll care more on how this change affects them, `.drop()` now accepts single columns, than how we implemented this internally (transforming the string to on element list). Do you mind rephrasing a bit please? Sorry I didn't see this in the previous review. Thanks!